.github/workflows/tests.yml

```diff
@@ -13,8 +13,8 @@ on:
   merge_group:
 
 jobs:
-  unit-tests:
-    name: "🛃 Unit tests"
+  unit-tests-f39:
+    name: "🛃 Unit tests (Fedora 39)"
```

Comment:
The reason I kept it id because github configuration on which test is required is based on name, which is annoying to do when a PR is open.  If I mark `Unit tests (Fedora 39)` as required now, other PRs wont be allowed to merge before this one because they wont have the F39 config.

Reply:
Changed the repo config to require the new name *and* Fedora 40.